invenio_notifications/templates/semantic-ui/invenio_notifications/email/community-invitation.accepted.html

```diff
@@ -0,0 +1,7 @@
+<p>{{ _('Invitation accepted') }}</p>
```

Comment:
minor/shelve: once the module is released we should move any community-specific templates to `invenio-communities`.

Reply:
will move templates in an upcoming PR